CMakeLists.txt

```diff
@@ -74,7 +74,9 @@ set(SOURCE_FILES
         src/shaka_scheme/system/base/Real.cpp
         src/shaka_scheme/system/base/Number.cpp
         src/shaka_scheme/system/gc/GCData.cpp
-        src/shaka_scheme/system/gc/GCList.cpp)
```

Comment:
This was bad -- it should be fixed after this. Core Systems should now be aware since I corrected Brandon's mistake on this formatting.